tests/translator/output/aws-cn/application_with_events.json

```diff
@@ -0,0 +1,71 @@
+{
+    "Resources": {
+      "Images": {
+        "Type": "AWS::S3::Bucket",
+        "DependsOn": [
+          "BasicApplicationFunctionInsideApplicationPermission"
+        ],
+        "Properties": {
+          "NotificationConfiguration": {
+            "LambdaConfigurations": [
+              {
+                "Function": {
+                  "Fn::Join": [
+                    "",
+                    [
+                      {
+                        "Fn::Sub": "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:"
```

Comment:
We are building this arn, right? the partition should be [${AWS::Partition}](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html) since it resolved to aws-cn for China regions and aws-gov for gov regions.